include/image_undistort/undistorter.h

```diff
@@ -36,8 +36,8 @@ class Undistorter {
  private:
   const CameraParametersPair used_camera_parameters_pair_;
 
-  cv::Mat map_x_;
-  cv::Mat map_y_;
+  cv::UMat map_x_;
+  cv::UMat map_y_;
 };
 }
 
```

Comment:
I think this is OpenCV3 which may be the default in 16.04, however, can we keep the Mat version in a different branch for the people still on 14.04?